controllers/configurationpolicy_controller.go

```diff
@@ -2280,26 +2282,19 @@ func (r *ConfigurationPolicyReconciler) checkAndUpdateResource(
 
 // AppendCondition check and appends conditions to the policy status
 func AppendCondition(
-	conditions []policyv1.Condition, newCond *policyv1.Condition, resourceType string, resolved ...bool,
+	conditions []policyv1.Condition, newCond *policyv1.Condition,
 ) (conditionsRes []policyv1.Condition) {
-	defer recoverFlow()
```

Comment:
I think this looks good! One final thought: @willkutler do you recall why this is here? It was put in 3 years ago, so definitely no worries if you're not sure! https://github.com/stolostron/config-policy-controller/commit/fbcef07151e2bb45843ee7c72d0267cb301a983a#diff-abd7bd58ac8e6e13efca328b26418fd3abf28bb9eec7112c669bbfd161e4e950
I want to make sure we're not introducing a panic somewhere, but I also don't recall seeing the "ALERT!" message in any logs...

Reply:
I don't unfortunately - some of the code was ported from the old policy controller so Yu might have written that. I haven't seen the ALERT message in any logs either so I think it's fine to remove

Reply:
Okay! Let's do this!!!